src/main/java/org/fcrepo/migration/foxml/ArchiveExportedFoxmlDirectoryObjectSource.java

```diff
@@ -56,8 +64,17 @@ public void setFetcher(final URLFetcher fetcher) {
         this.fetcher = fetcher;
     }
 
+    /**
+     * Sets a FileFilter to determine which files will be considered as object
+     * files in the source directories.
+     * @param fileFilter a FileFilter implementation
+     */
+    public void setFileFilter(final FileFilter fileFilter) {
```

Comment:
This method does not appear to ever be invoked (i.e. dead code). Shall we remove it?